workspaces/tech-radar/plugins/tech-radar/config.d.ts

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export interface Config {
+  techRadar?: {
+    /**
+     * Optional uri which will be hit to fetch the definition file for the tech radar entries. The file should be in JSON format.
+     * If both `proxyUri` & `entries` are specified, the `proxyUri` will be used.
+     * @visibility frontend
+     */
+    proxyUri?: string;
```

Comment:
This name is a little surprising to me - what exactly are we proxying here? I wonder if it's worth considering a single property with a union type that can accept a string URL or a data object.

Reply:
Happy to convert to a union, but left as two distinct named optionals since they feel like functionally different choices that have separate names/descriptors

Reply:
If we decide to go with the proxy option, I think we can skip this config. Usually, plugins use a 'well-known' proxy path. Here is an example: https://github.com/backstage/community-plugins/blob/main/workspaces/sentry/plugins/sentry/README.md?plain=1#L93

This means that you can decide on a hardcoded proxy config upfront, for example, `/tech-radar`, so that the plugin knows where to fetch the data.